src/app/shared/case-editor/case-edit-submit.component.ts

```diff
@@ -223,4 +223,20 @@ export class CaseEditSubmitComponent implements OnInit {
       .filter(cf => cf.show_summary_content_option);
   }
 
+  getCaseId(): String {
+    return (this.caseEdit.caseDetails ? this.caseEdit.caseDetails.case_id : '');
+  }
+
+  isEmail(field: CaseField): boolean {
```

Comment:
not sure I got what this method is doing. Why we need to call this.summaryCaseField(field)?

Reply:
I have fixed this by assigning it to a variable now.   I am calling this.summaryCaseField(field) to filter non-null fields.